File: gherkin/java/src/main/java/gherkin/events/PickleEvent.java
package gherkin.events;

import gherkin.pickles.Pickle;

public class PickleEvent implements CucumberEvent {
    private final String type = "pickle";
    public final String uri;
    public final Pickle pickle;

    public PickleEvent(String uri, Pickle pickle) {
        this.uri = uri;
        this.pickle = pickle;
    }
}

File: gherkin/dotnet/Gherkin/Events/PickleEvent.cs
using System.Runtime.Serialization;
using Gherkin.CucumberMessages.Types;

namespace Gherkin.Events
{
    public class PickleEvent : IEvent
    {
        [DataMember(Name = "pickle")]
        public Pickle EventArgs { get; set; }

        public PickleEvent(Pickle eventArgs)
        {
            EventArgs = eventArgs;
        }

        public PickleEvent()
        {
        }
    }
}
